Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM ruby:2.3.1
+FROM ruby:2.5.3
 
 ARG TIMEZONE
 ENV TIMEZONE=$TIMEZONE
 
 RUN echo $TIMEZONE > /etc/timezone
 RUN dpkg-reconfigure -f noninteractive tzdata
 
-RUN apt-get update -qq && apt-get install -y build-essential nodejs npm nodejs-legacy mysql-client vim openssh-client
```

Comment:
Removed npm as it caused this error in the CI

E: Unable to locate package npm
The command '/bin/sh -c apt-get update -qq && apt-get install -y build-essential nodejs npm nodejs-legacy mysql-client vim openssh-client' returned a non-zero code: 100
ERROR: Job failed: command terminated with exit code 100